.github/workflows/build.yml

```diff
@@ -61,11 +60,23 @@ jobs:
         run: |
           gcovr --sonarqube --exclude-throw-branches > coverage.xml
       - name: Run sonar-scanner
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          PR_HEAD_REF: ${{ github.event.pull_request.head.ref }}
+          PR_BASE_REF: ${{ github.event.pull_request.base.ref }}
         if: "${{ env.SONAR_TOKEN != '' }}"
         run: |
           sonar-scanner \
             --define sonar.cfamily.build-wrapper-output="${{ env.BUILD_WRAPPER_OUT_DIR }}" \
             --define sonar.coverageReportPaths=coverage.xml
+            --define sonar.pullrequest.github.repository=${{ github.event.pull_request.base.repo.full_name }} \
+            --define sonar.scm.revision=${{ github.event.pull_request.head.sha }}
```

Comment:
@GwnDaan Missing '\' on this line?

Reply:
Correct... Do you mind adding it to a PR somewhere? Or should we be making a dedicated PR for it?

Reply:
I can put it in #180